apps/backend/src/routes/login/+page.svelte

```diff
@@ -0,0 +1,161 @@
+<!-- JavaScript code -->
+<script lang="ts">
+  // import for forms
+  import { createForm } from 'svelte-forms-lib';
+
+  import { authStore } from '$lib/stores';
+  import type { FirebaseError } from 'firebase/app';
+
+  // import sign up function
+  import { signOut, signInEmailPassword } from '$lib/firebase/auth';
+
+  //import yup for form validation
+  import * as yup from 'yup';
+
+  let apiError: FirebaseError | null;
+
+  // Yup validation code
+  const { form, errors, handleChange, handleSubmit } = createForm({
+    initialValues: {
+      email: '',
+      password: '',
+    },
+    validationSchema: yup.object().shape({
+      email: yup.string().email().required('Please enter your Email.'),
+      password: yup.string().required('Please enter your Password.'),
+    }),
+
+    onSubmit: async (data) => {
+      console.log(data);
+      try {
+        await signInEmailPassword(data.email, data.password, (error) => (apiError = error));
+      } catch (error) {
+        console.error(error);
+      }
+    },
```

Comment:
Unless you require the `console.log` statement, you should be making the function point directly to `signInEmailPassword`.

E.g.,
```ts
onSubmit: (data) => signInEmailPassword(data.email, data.password, (error) => (apiError = error))
```

Note: Nothing will be caught in the `catch` statement because everything is handled in the `signInEmailPassword` api.

Reply:
Removed the console.log statements along with the unnecessary try logic.

Used the given code:
`onSubmit: (data) => signInEmailPassword(data.email, data.password, (error) => (apiError = error))`